docs/source/index.rst

```diff
@@ -46,7 +47,7 @@ be used anywhere where high-throughput and robust messaging are needed.
 Features
 ++++++++
 
-* Support for 1000s of messages per second
+* Support for `1000s of messages per second <performance>`__
```

Comment:
```suggestion
* Support for `thousands of messages per second <performance>`__
```
Think this is more correct and more legible but feel free to keep as is

Reply:
I'd like to keep the number here as I think it attracts the eye and this is a good point in the documentation to acts as an advertisment.